main-actions/src/main/scala/sbt/Pkg.scala

```diff
@@ -40,8 +40,6 @@ import xsbti.{ FileConverter, HashedVirtualFileRef, VirtualFile, VirtualFileRef
 object Pkg:
   def JarManifest(m: Manifest) = PackageOption.JarManifest(m)
   def MainClass(mainClassName: String) = PackageOption.MainClass(mainClassName)
-  def MainfestAttributes(attributes: (Attributes.Name, String)*) =
-    PackageOption.ManifestAttributes(attributes*)
```

Comment:
This should be split to another PR.

Reply:
This is because `Mainfest` is a lousy misspelling of `Manifest` and the default correction resulted in a conflict with the next function ...